content/docs/authorization/configuration/powerflex/_index.md

```diff
@@ -55,7 +55,7 @@ Given a setup where Kubernetes, a storage system, and the CSM for Authorization
 
     **Helm**
 
-    Refer to the [Install the Driver](../../../deployment/helm/drivers/installation/powerflex/#install-the-driver) section to edit the parameters in `samples/config.yaml` to configure the driver to communicate with the CSM Authorization sidecar.
+    Refer to the [Install the Driver](../../../deployment/helm/drivers/installation/powerflex.md/#install-the-driver) section to edit the parameters in `samples/config.yaml` to configure the driver to communicate with the CSM Authorization sidecar.
```

Comment:
@WilsonRadadia20 : why this change is required, we should not add extensions in hugo link

Reply:
I have removed the extensions.